### ISSUE TITLE
Error checking updates in main view

### ISSUE THREAD

**Issue by xKrypt0x** [Tobi823/ffupdater#243]
Stacktrace:
```
java.util.NoSuchElementException: Collection contains no element matching the predicate.
	at de.marmaro.krt.ffupdater.network.fdroid.FdroidConsumer.getLatestUpdate(Unknown Source:288)
	at de.marmaro.krt.ffupdater.network.fdroid.FdroidConsumer$getLatestUpdate$1.invokeSuspend(Unknown Source:12)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:94)
	at m4.e.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	at n4.a$a.run(SourceFile:9)
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.2.0 (113) release |
| Device | Pixel 5 (redfin, redfin, redfin) |
| Manufacturer | google (Google) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | grapheneos, grapheneos, release-keys, 1664826777000 |

**Comment by taylor-josh** [comment]
I got this error as well when trying to update Mull.

Looks like this is resolved in 77.3.0 according to the [changelog](https://github.com/Tobi823/ffupdater/releases/tag/77.3.0).

I haven't tested yet - was waiting for the update to hit FDroid.

**Comment by Tobi823** [comment]
> Looks like this is resolved in 77.3.0 according to the [changelog](https://github.com/Tobi823/ffupdater/releases/tag/77.3.0).

👍🏻
